MohammadBallal/utils.py

```diff
@@ -0,0 +1,27 @@
+#Function for validate the format of date
+def validate_date(datestr):
+    from datetime import datetime
+    try: 
+        #Validates if the date matches dd/mm/yyyy format by strptime() method 
+        datetime.strptime(datestr, '%d/%m/%Y')
+    except ValueError: 
+        raise ValueError("Date form must be dd/mm/yyyy")
+
+#Function to get valid details input by user    
+def get_valid_input(prompt):
+    userInput= input(prompt)
+    if not userInput:
+        raise ValueError("It's empty!")
+    return userInput
+
+#Function for validate status if its in the valid status list or not
+def validate_status(status):
+    validStatus= ['inprogress','completed']
```

Comment:
`self.status="InProgress"` in the constructor

Reply:
This issue was resolved after reviewing it with Samaheen.
its now changed in the code.